### PR TITLE
Updated analytics version, refactored AV1 demo to use v8 player

### PR DESCRIPTION
**Pull request by vzablock**
Issue: https://github.com/bitmovin/dashboard/issues/2944
Work: Updated analytics version on all player demos, refactored AV1 demo to use v8 player

### CODE REVIEW

File: encoding/multi-codec-streaming/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script type="text/javascript"
         src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
 <script type="text/javascript"
-        src="https://cdn.bitmovin.com/analytics/web/beta/2/bitmovinanalytics.min.js"></script>
+        src="https://cdn.bitmovin.com/analytics/web/2.5.2/bitmovinanalytics.min.js"></script>
```
Comment (Tigraine) [inline encoding/multi-codec-streaming/index.html]: Please stay with a latest /2

File: encoding/per-scene-adaptation/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <script type="text/javascript"
         src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
 <script type="text/javascript"
-        src="https://cdn.bitmovin.com/analytics/web/beta/2/bitmovinanalytics.min.js"></script>
+        src="https://cdn.bitmovin.com/analytics/web/2.5.2/bitmovinanalytics.min.js"></script>
```
Comment (Tigraine) [inline encoding/per-scene-adaptation/index.html]: Please don’t use a specific version here or we’ll need to update the demos all the time

**Comment by vzablock** [comment]
We discussed on slack taht it would be better to use `beta` since they use it for testing as well

File: player/stream-test/index.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
-<script type="text/javascript" src="https://cdn.bitmovin.com/analytics/web/1.7.2/bitmovinanalytics.min.js"></script>
+<script type="text/javascript" src="https://cdn.bitmovin.com/analytics/web/beta/2/bitmovinanalytics.min.js"></script>
```
Comment (lukaskroepfl) [inline player/stream-test/index.html]: not in favour of using it here as it could be broken

Comment (lukaskroepfl) [reply player/stream-test/index.html]: ...and this is the most visited page of bitmovin.com

**Pull request merged**